### PR TITLE
Adds an alarm for when the AME runs out of fuel

### PR DESCRIPTION
**Pull request by Zymem**
## About the PR 

**Changelog**

:cl: Zyme
- add: The AME will now warn you when it runs out of fuel

### CODE REVIEW

**Comment by Zymem** [comment]
Apparently there was an error on the tests when setting up .NET Core, more specifically when reading ECONNRESET...
Gotta be honest, I have no idea what any of that means, so any help would be appreciated.

**Comment by 0x6273** [comment]
This should preferably be ECSed also (all logic moved from component to system)

**Comment by Zymem** [comment]
I guess practicing C# PRs with old code probably wasn't the best idea, especially for learning ECS lmao.
I won't be having that much free time for the next couple of weeks, so I doubt I'll be trying to update the code to modern standards (also waay beyond my current scope).

I'll leave it up to the mantainers if they want to merge this PR, or prefer closing it until #11967 is solved.

File: Content.Server/AME/Components/AMEControllerComponent.cs [hunk 1]
```diff
@@ -285,6 +299,11 @@ private void InjectSound(bool overloading)
         {
             SoundSystem.Play(_injectSound.GetSound(), Filter.Pvs(Owner), Owner, AudioParams.Default.WithVolume(overloading ? 10f : 0f));
         }
+
+        private void NoFuelSound()
+        {
+            SoundSystem.Play(_noFuelSound.GetSound(), Filter.Pvs(Owner), Owner, AudioParams.Default.WithVolume(-2f));
```
Comment (EmoGarbage404) [inline Content.Server/AME/Components/AMEControllerComponent.cs:305]:
```suggestion
            _entities.EntitySysManager.GetEntitySystem<SharedAudioSystem>().PlayPvs(_noFuelSound, Owner, AudioParams.Default.WithVolume(-2f));
```
this is oldcode and not ECS, hence the super weird call here. Ideally this needs to be refactored, but it's better to not use obsolete methods in any case.

**Comment by Zymem** [comment]
Alright, thanks for the review. I'll probably just close the PR seeing as seeing as 0x6273 seems to be working on updating the AME code. If this isn't included in the refractor I'll try to see of adding it then.

**Pull request closed without merging**